jwst/pipeline/calwebb_ami3.py

```diff
@@ -10,14 +10,14 @@
 from ..ami import ami_analyze_step
 from ..ami import ami_average_step
 from ..ami import ami_normalize_step
-from ..resample import blend
+from ..model_blender import blendmeta
 
 __version__ = '0.8.0'
 
 # Define logging
 import logging
 log = logging.getLogger()
-log.setLevel(logging.DEBUG)
+log.setLevel(logging.INFO)
```

Comment:
Actually it's desired to keep this set at DEBUG level. We've discovered over the years that this setting acts as sort of a "floor" for the lowest level of messages that can/will be passed up by the routine. If it's set to INFO and the user requests DEBUG level, the DEBUG messages will NOT appear. If you set it here to DEBUG, the default level returned is INFO, but when the user requests DEBUG they'll actually show up.